### ISSUE TITLE
Teams Toolkit command line interface documentation has wrong parameters

### ISSUE THREAD

**Issue by appieschot** [MicrosoftDocs/msteams-docs#10372]
The documentation for [TeamsFx CLI v2 teamsfx package](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/teams-toolkit-cli?pivots=version-two#parameters-for-teamsfx-package) has wrong parameters. 

The documentation states: 
`--manifest-file-path | No | Select the Teams app manifest template path, defaults to ${folder}/appPackage/manifest.json.`

However, installing with `npm install @microsoft/teamsfx-cli@2.* -g` and then running `teamsfx package --help` returns: 

```bash
The 'teamsfx' command will be deprecated in the next major release. Start using `teamsapp` and Teams Toolkit CLI v3 (@microsoft/teamsapp-cli) to ensure capability with future updates. Learn more: https://aka.ms/teamsfx-toolkit-cli
Usage: teamsfx package [options]

Build your Microsoft Teams app into a package for publishing.

Options:
  --teams-manifest-file -t    Specifies the Microsoft Teams app manifest template file path, it can be either absolute
                              path or relative path to project root folder, defaults to './appPackage/manifest.json'
                              Default value: "./appPackage/manifest.json".
  --output-zip-path -oz       Specifies the output path of the zipped app package, defaults to
                              '${folder}/appPackage/build/appPackage.${env}.zip'.
  --output-manifest-path -om  Specifies the output path of the generated manifest path, defaults to
                              '${folder}/appPackage/build/manifest.${env}.json'
  --env                       Specifies the environment name for the project scaffolded by Microsoft Teams Toolkit.
  --folder -f                 Project folder. Default value: "./".

Global Options:
  --version -v                Display Microsoft Teams Toolkit CLI version.
  --help -h                   Show Microsoft Teams Toolkit CLI help.
  --interactive -i            Run the command in interactive mode. Default value: false.
  --debug                     Print debug information. Default value: false.
  --verbose                   Print diagnostic information. Default value: false.
  --telemetry                 Whether to enable telemetry. Default value: true.

For more information about the Teams Toolkit: https://aka.ms/teamsfx-cli.
```

So instead of `--manifest-file-path` it has to be `--teams-manifest-file`. A quick search for the other commands makes it looks as if the `--manifest-file-path` has to be `--teams-manifest-file` for all other commands as well.

**Comment by therealjohn** [comment]
Thanks! Fixed in #10376

**Comment by ChetanSharma-msft** [comment]
Hello @therealjohn - Thanks for raising the PR.
We will look into it and merge the changes.
